app/vmui/packages/vmui/src/pages/CardinalityPanel/CardinalityTips/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { TipIcon } from "../../../components/Main/Icons";
+import React, { FC } from "preact/compat";
+import { ReactNode } from "react";
+import "./style.scss";
+
+const Link: FC<{ href: string, children: ReactNode }> = ({ href, children }) => (
+  <a
+    href={href}
+    className="vm-link vm-link_colored"
+  >
+    {children}
+  </a>
+);
+
+const TipCard: FC<{ title?: string, children: ReactNode }> = ({ title, children }) => (
+  <div className="vm-cardinality-tip">
+    <div className="vm-cardinality-tip-header">
+      <div className="vm-cardinality-tip-header__tip-icon"><TipIcon/></div>
+      <h4 className="vm-cardinality-tip-header__title">{title || "Tips"}</h4>
+    </div>
+    <p className="vm-cardinality-tip__description">
+      {children}
+    </p>
+  </div>
+);
+
+export const TipDocumentation: FC = () => (
+  <TipCard title="Cardinality explorer">
+    <h6>Helpful for analyzing VictoriaMetrics TSDB data</h6>
+    <ul>
+      <li>
+        <Link href="https://docs.victoriametrics.com/#cardinality-explorer">
+          Cardinality explorer documentation
+        </Link>
+      </li>
+      <li>
+        See the <Link href="https://victoriametrics.com/blog/cardinality-explorer/">
+        example of using</Link> the cardinality explorer
+      </li>
+    </ul>
+  </TipCard>
+);
+
+export const TipHighNumberOfSeries: FC = () => (
+  <TipCard title="Metrics with a high number of series">
+    <ul>
+      <li>
+        You can to identify and eliminate labels with frequently changed values to reduce its&nbsp;
```

Comment:
```suggestion
        Identify and eliminate labels with frequently changed values to reduce their&nbsp;
```

Reply:
Can we make this and rest of links to open in a new tab? It would be unfortunate to lose the progress on the page just because of clicking on the link.